cmd/config-gen/utils/default.go

```diff
@@ -89,5 +89,25 @@ func DefaultConfig(rateInterval time.Duration, labelPrefix string) *MetricsDisco
 				MetricsQuery: fmt.Sprintf("sum(rate(<<.Series>>{<<.LabelMatchers>>}[%s])) by (<<.GroupBy>>)", pmodel.Duration(rateInterval).String()),
 			},
 		},
+
+		ResourceRules: &ResourceRules{
+			CPU: ResourceRule{
+				ContainerQuery: fmt.Sprintf("sum(rate(container_cpu_usage_seconds_total{<<.LabelMatchers>>}[%s])) by (<<.GroupBy>>)", pmodel.Duration(rateInterval).String()),
```

Comment:
I'm curious about this, do you think for "node" metrics, should we sum up the container usage or actually use the node-exporter's metrics. The latter would probably be the more correct one I would say, but induce an additional dependency.

Reply:
I'm matching what the summary API does right now, which appears to be looking at the root cgroup.

Reply:
(which I do think makes some sense)